website/docs/r/storage_account.html.markdown

```diff
@@ -81,6 +83,14 @@ The following arguments are supported:
 
 ~> **Note:** [More information on Validation is available here](https://docs.microsoft.com/en-gb/azure/storage/blobs/storage-custom-domain-name)
 
+---
+
+`identity` supports the following:
+
+* `type` - (Required) Specifies the identity type of the Storage Account. At this time the only allowed value is `SystemAssigned`.
+
+~> The assigned `principal_id` and `tenant_id` can be retrieved after the Storage Account has been created. More details are available below.
```

Comment:
Might it be worth explaining these properties won't be assigned until the identity type value is set to `SystemAssigned` ?